Automattic/ElasticPress
# Update Elasticpress to 3.6.1

## Description

We had a big PR when we merged upstream into this repository to match Elasticpress 3.5.6: https://github.com/Automattic/ElasticPress/pull/96

This PR does the same procedure but to update to 3.6.1. We had **way less** conflicts since the changes from that version are not that big.

**Please test this branch on a Sandbox with non-trivial WP installation.**

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Perform some searches on the front-end.
3. CRUD some posts and see that the results appear without errors on searches.
